src/data.js

```diff
@@ -1699,6 +1699,28 @@ module.exports = [
       'JavaScript',
       'Accessibility',
     ],
+  },
+    {
+    name: 'Ethan Aldrich',
+    description:
+      'CTO of SSE, full-stack developer, student.,
```

Comment:
Can you try removing the period on "student"?

Reply:
There's no closing quotation mark, hence build is failing.